### ISSUE TITLE
Installation path does not exist

### ISSUE THREAD

**Issue by yann2-0** [jorich-2000/simplesaml-joomla#2]
Hello, when i install the zip in joomla package installation error where is this path? thank you

**Comment by jorich-2000** [comment]
If you download from github then you just get the raw source code - see below how to create an installable package**
There is working Joomla package file you can download here hhttps://oldskoolhouse.co.uk/wordpress/category/utilities/

** To create installable Joomla package from source
1- unzip the downloaded zip file
2- zip the simplesamlsso folder as plg_simplesamlsso.zip and save in simplesamlsso folder - you can remove the rest of the files in this folder now
3 - zip the simplesamlslo folder as plg_simplesamlslo.zip and save in simplesamlslo folder- you can remove the rest of the files in this folder now
4 - now zip the whole simplesamlauth folder as simplesamlauth.zip including the

**Comment by yann2-0** [comment]
Thank you for your reply. successful installation but when I try to connect the error message is '' PLG_SIMPLESAML_CONFIG_DIR_UNDEFINED '' in the attribute mapping tab usually I just have username and password to enter. Sorry I'm a beginner in this field

**Comment by yann2-0** [comment]
I succeeded with the environment variable however it still doesn't work when I try to connect it sends me back to the simpleaml page and when I re-enter the identifiers I get the following error:

SimpleSAML \ Error \ Error: UNHANDLEDEXCEPTION

Backtrace:
1 www / _include.php: 17 (SimpleSAML_exception_handler)
0 [builtin] (N / A)
Caused by: Exception: The POST data we should restore was lost.
Backtrace:
2 modules / core / www / postredirect.php: 38 (require)
1 lib / SimpleSAML / Module.php: 260 (SimpleSAML \ Module :: process)
0 www / module.php: 10 (N / A)

**Comment by jorich-2000** [comment]
Have you successfully used the simplesaml page to test your authentication sources?

Jonathan
________________________________

**Comment by yann2-0** [comment]
yes my source of identification is working successfully. If I enter an identifier that does not exist in the simpleaml active directory tells me that the user does not exist. When I enter an existing user, he gives me the previous error message

**Comment by jorich-2000** [comment]
That error looks like it is coming from the SimpleSAMLPHP component. Can 
you attach a screen shot of the page the error is seen on


If simplesaml is working correctly then you should be able to browse to 
https://yourdmain.tld/simplesaml and see something like


Use the 'Authentication' tab to check you can log in to your defined 
authentication source - that should respond with a list of SAML 
attributes and groups.


Once you are sure that is all working you can use the attribute names 
from this screen to populate the fields in the Joomla component
(out of interest what identity provider are you connecting too?)

Jonathan

------ Original Message ------

**Comment by yann2-0** [comment]
![Capture d’écran 2021-02-03 à 11 15 49](https://user-images.githubusercontent.com/59530991/106711650-400cf400-6611-11eb-9101-75f736a7a84f.png)
![Capture d’écran 2021-02-03 à 11 08 36](https://user-images.githubusercontent.com/59530991/106711675-48fdc580-6611-11eb-813b-fffcd72a190c.png)

**Comment by yann2-0** [comment]
my source is a ldap when j enter a bad simpleaml identifier di me di although the identifier is not correct I don't see what information you want me to enter in the joomla component? sorry I'm a beginner

**Comment by jorich-2000** [comment]
Before doing anything in Joomla you must get SimpleSAML installed and 
working - see 
https://simplesamlphp.org/docs/stable/simplesamlphp-install
In simplesaml you will configure an authentication source to your ldap 
idp - see SimpleSAMLphp Identity Provider QuickStart 
<https://simplesamlphp.org/docs/stable/simplesamlphp-idp>
With your idp configured you can use the 'Federation' tab of the 
SimpleSAMLPHP install on your website (it should be accessible via 
https://YOURDOMAN.TLD/simplesaml) to check the Idp - you can also use 
the Authentication tab to check that your are getting authenticated 
correctly - I cant stress enough get SimpleSAML running before you do 
anything in Joomla.
Once you are able to authenticate in SimpleSAML send me a screen shot of 
the SimpleSAML Authentication and Federation pages and I will tell you 
what you need to enter in Joomla

Jonathan

------ Original Message ------

**Comment by yann2-0** [comment]
this is already the case here is again a capture of the simpleaml successful authentication
![Capture d’écran 2021-02-03 à 14 06 09](https://user-images.githubusercontent.com/59530991/106731749-5030cd80-6629-11eb-9c59-57052a4def8a.png)

**Comment by yann2-0** [comment]
![Capture d’écran 2021-02-03 à 14 22 32](https://user-images.githubusercontent.com/59530991/106733410-4a3bec00-662b-11eb-8d55-36f61551682b.png)

**Comment by yann2-0** [comment]
![Capture d’écran 2021-02-03 à 15 26 36](https://user-images.githubusercontent.com/59530991/106740743-3e085c80-6634-11eb-9d8e-cc5cc3d063c4.png)

**Comment by yann2-0** [comment]
any response ?
